openquake/server/templates/engine/get_outputs_impact.html

```diff
@@ -35,6 +35,11 @@ <h2>Outputs from calculation {{ calc_id }}: {{ description }}{% if local_timesta
                 <a href="{{ oq_engine_server_url }}/v1/calc/{{ calc_id }}/download_png/assets.png" class="btn btn-sm">
                   Show assets</a>
               </div>
+              {% if user.is_superuser %}
```

Comment:
Thinking a bit more, I want to show the page to the risk team, so let's make it visible to everybody for the moment. There is nothing secret.

Reply:
I can make the button visible to level 2 users

Reply:
Perfect